client/src/js/references/components/Detail/Targets/Targets.js

```diff
@@ -78,6 +78,12 @@ export class Targets extends React.Component {
             );
         }
 
+        const none = (
+            <BoxGroupSection>
```

Comment:
- use `NoneFoundSection` base component
- need to provide `noun` prop ('targets' should be value)
- why define `none` when you aren't going to render it? This is the better way to do it:
```js
let noneFound

if (!targetComponents.length) {
  noneFound = <NoneFoundSection noun="targets">
}
```